src/dex-ui/pages/Governance/formatter.ts

```diff
@@ -34,7 +34,7 @@ export const formatProposal = (proposal: Proposal): FormattedProposal => {
       yes,
       no,
       abstain,
-      quorum,
+      quorum: (Number(quorum) / Number(max)) * 100,
```

Comment:
We should use the BigNumber library for decimal arithmetic to avoid precision errors. Please update the older calculation for `remaining` as well.

Reply:
So I tried to do the `arithmetic` using `BigNumber` Library, one UI concern which I saw that after `arithmetic` operation on binary if we try to convert it number to show in UI the number is too big after decimal point and the `quorum` value is too small. please check the screenshot below.
let me know how should be proceed with that.

![Screenshot 2023-02-09 at 6 04 27 PM](https://user-images.githubusercontent.com/89677167/218008401-6cd00b6b-92f7-46b0-8347-3eb6f01a66be.png)

![Screenshot 2023-02-09 at 6 04 24 PM](https://user-images.githubusercontent.com/89677167/218008340-9206a90e-1175-4d7c-842e-70ab161e3101.png)

Reply:
Quorum can be rounded to the closest integer percent value. The vote count rounding should match what is shown on the proposal details page.

Reply:
Done